content/en/apps/features/integrations/openmrs.md

```diff
@@ -4,6 +4,12 @@ weight: 2
 description: >
    Exchange patient-level data with systems based on the OpenMRS platform
 keywords: openmrs
+relatedContent: >
+  apps/features/integrations/generic
+  apps/features/integrations/rapidpro
+  apps/features/integrations/dhis2
 ---
 
+[OpenMRS](https://openmrs.org) is an open source electronic medical record system used in dozens of countries. Integrating CHT apps with OpenMRS can open up opportunities to improve health outcomes for patients by promoting better coordination of care. For example, referrals by CHWs in the community can be sent electronically to health facilities using OpenRMS so that nurses and clinicians can prepare for their visit and have full access to the assessment done by a CHW.
```

Comment:
```suggestion
[OpenMRS](https://openmrs.org) is an open source electronic medical record system used in dozens of countries. Integrating CHT apps with OpenMRS can open up opportunities to improve health outcomes for patients by promoting better coordination of care. For example, referrals by CHWs in the community can be sent electronically to health facilities using OpenMRS so that nurses and clinicians can prepare for their visit and have full access to the assessment done by a CHW.
```

Reply:
Corrected a typo, but otherwise seems fine as a placeholder